stubs/python-jenkins/jenkins/__init__.pyi

```diff
@@ -92,6 +92,14 @@ class WrappedSession(requests.Session):
 _JSONValue: TypeAlias = Any  # too many possibilities to express
 _JSON: TypeAlias = dict[str, _JSONValue]
 
+@type_check_only
+class _Job(TypedDict, total=False):
+    _class: str
+    url: str
+    color: str
+    name: str
```

Comment:
It seems the `fullname` attribute was missing, moreover it seems required
and `name` also seems required
```suggestion
    name: Required[str]
    fullname: Required[str]
```